### PR TITLE
CDAP-2221: RealtimeCubeSink implementation

### PR DESCRIPTION
**Pull request by abaranau**

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sinks/RealtimeCubeSink.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    return count;
+  }
+
+  @Override
+  public void initialize(RealtimeContext context) throws Exception {
+    super.initialize(context);
+    Map<String, String> runtimeArguments = context.getRuntimeArguments();
+    datasetName = runtimeArguments.get(NAME_PROPERTY);
+    transform = new StructuredRecordToCubeFact(runtimeArguments);
+
```
Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sinks/RealtimeCubeSink.java]: nit: remove extra line

**Comment by albertshau** [comment]
one very minor comment, then lgtm

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DT52-1

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DT52-5 GREEN

**Pull request merged**